### ISSUE TITLE
Why is the alphabet in that order - is it because of that song?

### ISSUE THREAD

**Issue by paulp** [lihaoyi/Ammonite#42]
A typical completion presentation. The docs say "you get the same autocomplete-anywhere support that you get in a modern IDE." I haven't checked lately but I bet the modern IDE alphabetizes.
```
##                 asInstanceOf       isInstanceOf       toString           hashCode           equals
!=                 ==                 size               isEmpty            elemAt             foreach
getClass           foreachSlice       viewOps            mapIndices         foreachIntIndex    foreachIndex
containsIndex      lastNth            lastIndex          lastIntIndex       nths               indices
isPositive         isZero             intSize            longSize           byShow             byRef
byEquals           partition          span               splitAt            join               directlySlice
linearlySlice      build              force              withFilter         takeWhile          takeRight
take               map                flatMap            dropWhile          dropRight          drop
collect            ++                 :|                 |:                 xs                 toEach
foldFrom           fold               foldr              foldl              foldWithIndex      groupBy
clusterBy          dropIndex          takeToFirst        ofClass            zipView            zipIndex
zip                withSize           tail               sliceWhile         slice              mapZip
mapWithIndex       mapApply           labelOp            init               grep               gatherClass
gather             filterNot          filter             sortBy             sortDistinct       sorted
min                max                zfoldr             zfoldl             zfirst             toRefs
tabular            nonEmpty           mk_s               mkString           last               indicesWhere
indexWhere         head               forallTrue         forall             first              find
exists             count              foreachReverse     foreachWithIndex   synchronized       ne
eq                 wait               notifyAll          notify             any_s              try_s
matchOr            requiring          optionally         same               fix                id_##
id_==              |>                 sideEffect         isOr               doto               reflect
isNull             toRef              castTo             isClass            shortPackage       shortClass
x                  __shown_rep
```

**Comment by lihaoyi** [comment]
IntelliJ actually does clever things in detecting which methods you use the most and puts those at the top. In this case, Ammonite has clearly looked at your Scala REPL history and found that you use `##` the most out of any completion. You must do a lot of casting too!

**Comment by lihaoyi** [comment]
Fixed by cd64d736d6d3f15b9de7d263d20f8d66bbca2a21

```
@ Seq().Display all 182 possibilities? (y or n)
!=                   ##                   +                    ++
++:                  +:                   ->                   /:
:+                   :\                   ==                   WithFilter
addString            aggregate            andThen              apply
applyOrElse          asInstanceOf         canEqual             collect
collectFirst         combinations         companion            compose
contains             containsSlice        copyToArray          copyToBuffer
corresponds          count                diff                 distinct
drop                 dropRight            dropWhile            endsWith
ensuring             eq                   equals               exists
filter               filterNot            find                 flatMap
flatten              fold                 foldLeft             foldRight
forall               foreach              formatted            genericBuilder
getClass             groupBy              grouped              hasDefiniteSize
hashCode             head                 headOption           indexOf
indexOfSlice         indexWhere           indices              init
inits                intersect            isDefinedAt          isEmpty
isInstanceOf         isTraversableAgain   iterator             last
lastIndexOf          lastIndexOfSlice     lastIndexWhere       lastOption
length               lengthCompare        lift                 map
max                  maxBy                min                  minBy
mkString             ne                   nonEmpty             notify
notifyAll            orElse               padTo                par
partition            patch                permutations         prefixLength
product              reduce               reduceLeft           reduceLeftOption
reduceOption         reduceRight          reduceRightOption    repr
reverse              reverseIterator      reverseMap           runWith
sameElements         scan                 scanLeft             scanRight
segmentLength        seq                  size                 slice
sliding              sortBy               sortWith             sorted
span                 splitAt              startsWith           stringPrefix
sum                  synchronized         tail                 tails
take                 takeRight            takeWhile            to
toArray              toBuffer             toIndexedSeq         toIterable
toIterator           toList               toMap                toParArray
toSeq                toSet                toStream             toString
toTraversable        toVector             transpose            union
unzip                unzip3               updated              view
wait                 withFilter           zip                  zipAll
zipWithIndex         →
```
